loader-tests/check-opml.ts

```diff
@@ -0,0 +1,97 @@
+import './dom-parser.js'
```

Comment:
I rename file to fit with script name (and also to explain what exact type of test we will have here).

Since we are going to have many tests, none of them are main, so we can’t use `index.ts`.